### DISCUSSION TITLE
Transfer Limits

### DISCUSSION THREAD

**Discussion by Mohamed-Emara77** [supabase/supabase#2068] [Questions] [answered]
I would like to know if basic data access (ex : authentication requests) are counted inside Transfer Limits or not.

thanks

**Comment by inian** [comment, marked as answer]
Hi @Mohamed-Emara77, only storage APIs count for the transfer limits according to the current pricing plan.

**Reply by Mohamed-Emara77** [reply]
Thank you for that ❤️

**Reply by zidkim** [reply]
The "Transfer limits" show in two places on the [pricing page](https://supabase.com/pricing) -- Database and Object storage.
Are you saying download operations only refer to storage API calls in both/every scenario?
